staging/src/k8s.io/apiserver/pkg/cel/library/quantity.go

```diff
@@ -0,0 +1,122 @@
+package library
+
+import (
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/common/types"
+	"github.com/google/cel-go/common/types/ref"
+	"k8s.io/apimachinery/pkg/api/resource"
+	apiservercel "k8s.io/apiserver/pkg/cel"
+)
+
+func Quantity() cel.EnvOption {
```

Comment:
We should add library documentation as the godoc of this function, in the style of CEL library documentation (e.g. https://github.com/kubernetes/kubernetes/blob/ba2d98f5e5cc88c2ffbface0b1c0a999347565c7/staging/src/k8s.io/apiserver/pkg/cel/library/lists.go#L29).

Reply:
added doc